gpii/node_modules/matchMakerFramework/src/MatchMakerFramework.js

```diff
@@ -63,7 +63,22 @@
      * initialPayload consists of:
      *    userToken, preferences, deviceContext, solutionsRegistryEntry
      */
-    gpii.matchMakerFramework.preProcess = function (that, initialPayload) {
+    gpii.matchMakerFramework.preProcess = function (that, lifecycleManager, initialPayload) {
+        var activeTokens = lifecycleManager.activeUserTokens;
+        if (activeTokens.length === 1) {
```

Comment:
Hmm...  yeah, I can see how this is cryptic... The 'preProcess' request contains a userToken itself, and if we already have a user logged into the system, we modify the 'initialPayload' to be a request for accomodating both of those users. And the idea is that we will never get to this point attempting to preprocess with a third user (this is done via the user UserLogonStateChange file). I do see potential issues when we get further on the rematching work -- on the other hand, it weems wrong to introduce all sorts of logic here, dealing with the amount of users logged in, deciding whether some should be logged out, etc. And as it stands for this review, we only support two simoultaneous users